src/main/php/webservices/rest/RestRequest.class.php

```diff
@@ -281,7 +281,7 @@ public function withSegment($name, $value) {
    * @return  peer.Header
    */
   public function addHeader($arg, $value= null) {
-    if ($arg instanceof Header) {
+    if ($arg instanceof \peer\Header) {
```

Comment:
There already is an include for `peer\http\Header`, not sure it is correct what you are doing here.